packages/flutter/lib/src/gestures/multidrag.dart

```diff
@@ -17,12 +18,11 @@ typedef Drag GestureMultiDragStartCallback(Point position);
 
 /// Interface for receiving updates about drags from a [MultiDragGestureRecognizer].
 abstract class Drag {
-  /// The pointer has moved by the given offset.
-  void move(Offset offset) { }
+  /// The pointer has moved.
+  void update(DragUpdateDetails details) { }
 
-  /// The pointer is no longer in contact with the screen and was moving at a
-  /// given velocity when it stopped contacting the screen.
-  void end(Velocity velocity) { }
+  /// The pointer is no longer in contact with the screen.
```

Comment:
We should probably mention that you can get the velocity from the details. Similar comments in other dartdocs in this PR.